tests/test_create_inconsistent_dimensionality.py

```diff
@@ -0,0 +1,55 @@
+"""
+When a "context" passed to shape/asShape has a coordinate
+which is missing a dimension we should raise a descriptive error.
+
+When we use mixed dimensions in a WKT geometry, the parser strips
+any dimension which is not present in every coordinate.
+"""
+
+
+import pytest
+from pytest import mark
```

Comment:
I've gotten in the habit of following the usage in the pytest docs. Would you be okay with always using the fully qualified name like `@pytest.mark.parametrize()`?

Reply:
Sure, no problem.